### PR TITLE
CNV-32032: Release note deprecated feature

### PR DESCRIPTION
**Pull request by aspauldi**
Version(s): 4.14

Issue: [CNV-32032](https://issues.redhat.com/browse/CNV-32032)

Link to docs preview: [Release note: Deprecated feature Windows 2012 R2 templates](https://63947--docspreview.netlify.app/openshift-enterprise/latest/virt/release_notes/virt-4-14-release-notes#virt-4-14-deprecated)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by aspauldi** [comment]
@stu-gott, Can you please review this release note? Thanks

**Comment by aspauldi** [comment]
Updated to remove mention of Windows 2012, per Kedar.

**Comment by stu-gott** [comment]
LGTM

**Comment by aspauldi** [comment]
Hi @kbidarka@redhat.com, can you review this PR for me? Thanks!

**Comment by kbidarkar** [comment]
LGTM

File: virt/release_notes/virt-4-14-release-notes.adoc [hunk 1]
```diff
@@ -117,6 +117,9 @@ Deprecated features are included in the current release and supported. However,
 
 //CNV-29048 Release note: DEPRECATED FEATURE (Metrics backlog 4.14)
 
+//CNV-32032 Release note: DEPRECATED FEATURE (Windows 2012R2 templates deprecated)
+* Support for Windows Server 2012R2 templates is deprecated.
```
Comment (mburke5678) [inline virt/release_notes/virt-4-14-release-notes.adoc:121]: It looks like there is [supposed to be a space](https://www.microsoft.com/en-us/evalcenter/evaluate-windows-server-2012-r2):

```suggestion
* Support for Windows Server 2012 R2 templates is deprecated. 
```

**Comment by mburke5678** [comment]
@aspauldi One tiny nit. Otherwise LGTM

**Pull request merged**